### DISCUSSION TITLE
Hyphen in logins are not accepted!

### DISCUSSION THREAD

**Discussion by sogln** [Leantime/leantime#521] [General]
Bug:
a.xxx@t-online.de 
The hyphen "-" is not accepted for the login!

Regards
Christian

**Comment by marcelfolaron** [comment]
Ah, thanks. I'll make this an issue.

**Reply by danshearer** [reply]
In which case you should perhaps mention in the docs that hyphens are not valid in mysql/mariadb, because people often try to keep the two the same or consistent.
